### ISSUE TITLE
Tooltip not positioned correctly

### ISSUE THREAD

**Issue by IWRichard** [zurb/foundation-sites#7917]
I made a set of tooltips using Foundation 6, but out of each 4 tooltips, 1 is in the wrong position.

All tooltips are made as such:

```html
<a class="fa fa-fw fa-play has-tip left" href="#" data-tooltip aria-haspopup="true" title="Start"></a>
<a class="fa fa-fw fa-stop has-tip left" href="#" data-tooltip aria-haspopup="true" title="Stop"></a>
<a class="fa fa-fw fa-refresh has-tip left" href="#" data-tooltip aria-haspopup="true" title="Restart"></a>
<a class="fa fa-fw fa-trash has-tip left" href="#" data-tooltip aria-haspopup="true" title="Delete"></a>
```

As you can see, I'm explicitly saying a left tooltip.

But the resulting tooltip HTML is:

```html
<div class="tooltip left" role="tooltip" aria-hidden="true" data-is-active="false" data-is-focus="false" id="9fdgrf-tooltip" style="display: none; top: 266.969px; left: 1238.94px; opacity: 1;">Start</div>
<div class="tooltip top" role="tooltip" aria-hidden="true" data-is-active="false" data-is-focus="false" id="hnvv1j-tooltip" style="display: none; top: 219.469px; left: 1367.16px; opacity: 1;">Stop</div>
<div class="tooltip left" role="tooltip" aria-hidden="true" data-is-active="false" data-is-focus="false" id="gbgjxc-tooltip" style="display: none; top: 266.969px; left: 1316.22px;">Restart</div>
<div class="tooltip left" role="tooltip" aria-hidden="true" data-is-active="false" data-is-focus="false" id="7ao4ax-tooltip" style="display: none;">Delete</div>
```

Most notably is the second one: `<div class="tooltip top" role="tooltip" aria-hidden="true" data-is-active="false" data-is-focus="false" id="hnvv1j-tooltip" style="display: none; top: 219.469px; left: 1367.16px; opacity: 1;">Stop</div>` where if you look at the class it'll say **top** instead of **left** as expected.

Why is this?

**Comment by zurbchris** [comment]
yes, :laughing: regex is matching `top` from `stop`. I'll update the regex.

**Comment by zurbchris** [comment]
fixed, typo in my commit message didn't link it... 23fe3c

**Comment by IWRichard** [comment]
Thank you! :+1: 
When can I expect this fix to be released in the next 6.1.x version?
